CHANGELOG.md

```diff
@@ -1,122 +1,124 @@
+- We've added the `AllowUnresolvedParameters` function flag that disables warnings for processing RPCs with unresolved parameters. To enable this flag, use Blueprints, or add a tag to the `UFUNCTION` macro.
+- There is now a warning if you launch a cloud deployment with the `manual_worker_connection_only` flag set to true.
+- We now support server travel for single server game worlds. Does not currently support zoning or off-loading.
+- We've improved the workflow relating to schema generation issues and launching local builds. There is now a warning if you try to run a local deployment after a schema error.
+- `DeploymentLauncher` can now launch a simulated player deployment independently from the target deployment.
 Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-deployment-name> <sim-deployment-name> <sim-deployment-json> <sim-deployment-region> <num-sim-players> <auto-connect>`
 - Added `HeartbeatTimeoutWithEditorSeconds` and use it if WITH_EDITOR is defined to prevent workers disconnecting when debugging while running in editor.
-- Added the `bAsyncLoadNewClassesOnEntityCheckout` setting to SpatialGDKSettings that allows loading new classes asynchronously when checking out entities. This is off by default.
-- Added `IndexYFromSchema` functions for the `Coordinates`, `WorkerRequirementSet`, `FRotator`, and `FVector` classes. Remapped the `GetYFromSchema` functions for the same classes to invoke `IndexYFromSchema` internally, in line with other implementations of the pattern.
-- The logic responsible for taking an Actor and generating the array of Components that represents it as an Entity in SpatialOS has been extracted into `EntityFactory`.
-- Clients will now validate schema against the server and log a warning if they do not match.
-- Entries in the SchemaDatabase are now sorted to improve efficiancy when browsing the asset in the editor. (DW-Sebastien)
-- Load Balancing Strategies and Locking Strategies can be set per-level using SpatialWorldSettings.
+- We've added the `bAsyncLoadNewClassesOnEntityCheckout` setting to SpatialGDKSettings that allows loading new classes asynchronously when checking out entities. This is off by default.
+- We've added `IndexYFromSchema` functions for the `Coordinates`, `WorkerRequirementSet`, `FRotator`, and `FVector` classes. We've remapped the `GetYFromSchema` functions for the same classes to invoke `IndexYFromSchema` internally, in line with other implementations of the pattern.
+- Clients now validate schema against the server, and log a warning if they do not match.
+- Entries in the schema database are now sorted to improve efficiency when searching for assets in the Unreal Editor.
 - Batch Spatial Position Updates now defaults to false.
```

Comment:
I couldn’t find this in the docs, and I don’t know what sort of thing it is (and therefore how to format/capitalise it).

Reply:
Same question as above - should we mention that this is in the SpatialOS Settings window? (Also, can you confirm that the setting is phrased exactly like that - `Batch Spatial Position Updates`?)

Reply:
I prefer the settings class mention

Reply:
OK, that's fine.

Can you confirm that the setting is phrased exactly like that - Batch Spatial Position Updates?

Also, see my earlier comment about `GDKSettings` vs `SpatialGDKSettings` vs `SpatialGDKSettings.ini`.

Reply:
`BatchSpatialPositionUpdates`
`SpatialGDKSettings`